agent/command/s3_put_test.go

```diff
@@ -494,24 +493,29 @@ func TestPreservePath(t *testing.T) {
 	require.NoError(t, s.Execute(ctx, comm, logger, conf))
 	it, err := s.bucket.List(ctx, "")
 	require.NoError(t, err)
+
 	expected := map[string]bool{
-		"remote/foo":                            false,
-		"remote/myWebsite/assets/asset1":        false,
-		"remote/myWebsite/assets/asset2":        false,
-		"remote/myWebsite/assets/asset3":        false,
-		"remote/myWebsite/assets/images/image1": false,
-		"remote/myWebsite/assets/images/image2": false,
+		filepath.Join("remote", "foo"):                                     false,
+		filepath.Join("remote", "myWebsite", "assets", "asset1"):           false,
+		filepath.Join("remote", "myWebsite", "assets", "asset2"):           false,
+		filepath.Join("remote", "myWebsite", "assets", "asset3"):           false,
+		filepath.Join("remote", "myWebsite", "assets", "images", "image1"): false,
+		filepath.Join("remote", "myWebsite", "assets", "images", "image2"): false,
 	}
-	grip.Infof("expected: %s", expected)
-	grip.Infof("received: ")
-	for it.Next(ctx) {
-		grip.Infof("\n: %s", it.Item().Name())
 
+	for it.Next(ctx) {
 		expected[it.Item().Name()] = true
 	}
 
-	for item, exists := range expected {
-		require.True(t, exists, item)
+	trueForExpected := true
+	trueForExpectedWindows := true
+
+	for _, exists := range expected {
+		if !exists {
+			trueForExpected = false
```

Comment:
Rather than define a variable to check after the loop, can we just `assert.True(t, exists, <the file name that should have existed>)`? Would also have the benefit of telling exactly which files were expected but nonexistent.

Reply:
done